src/scala/scripts/ScroogeWorker.scala

```diff
@@ -86,10 +73,19 @@ object ScroogeWorker extends Worker.Interface {
       sys.error("onlyTransitiveThriftSrcs and immediateThriftSrcs should " +
         s"have not intersection, found: ${intersect.mkString(",")}")
 
+    // To preserve current default behaviour.
+    val defaultConfig = ScroogeConfig(
+      destFolder = scroogeOutput.toString,
+      includePaths = onlyTransitiveThriftSrcJars ++ remoteSrcJars,
+      // always add finagle option which is a no-op if there are no services
```

Comment:
I wonder if this default is needed when this flag can be passed in `compiler_args` field in **thrift_library** rule?

Reply:
People could pass it from outside, but flipping the default value from true to false would be a semantic (and probably breaking) change.
I think that's a great avenue to explore if you're interested, but I'd probably do it as a follow-up, and not part of this big diff.